src/redux/Books/Books.js

```diff
@@ -0,0 +1,26 @@
+// actions
+const ADD_BOOK = 'ADD_BOOK';
+const REMOVE_BOOK = 'REMOVE_BOOK';
+
+// reducer
+const booksReducer = (state = [], action) => {
+  switch (action.type) {
+    case ADD_BOOK:
+      return 'add a book';
```

Comment:
- [ ] In your reducer, you should be adding a new book to your state and not simply returning a string called `add a book`. Kindly rectify this :+1: